src/components/newsArchive.js

```diff
@@ -0,0 +1,70 @@
+import React from "react"
+import { graphql, useStaticQuery } from "gatsby"
+import "./newsArchive.scss"
+
+import NewsItem from "./newsItem"
+
+export default function NewsArchive() {
+  const data = useStaticQuery(graphql`
+    {
+      allContentfulNewsItem(
+        sort: { fields: date, order: DESC }
+        filter: { title: { ne: "Why #ArtsMatter" } }
+      ) {
+        edges {
+          node {
+            title
+            summary {
+              summary
+            }
+            contentful_id
+            date
+            headerImage {
+              fixed {
+                src
+              }
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  const newsItems = data.allContentfulNewsItem.edges.reduce((news, curNews) => {
+    const { date, title, contentful_id } = curNews.node
+    const summary =
+      curNews.node.summary === null ? null : curNews.node.summary.summary
+    const imgSrc =
+      curNews.node.headerImage !== null
+        ? curNews.node.headerImage.fixed.src
+        : null
+    const newNewsItem = {
+      title,
+      date,
+      contentfulId: contentful_id,
+      summary,
+      imgSrc,
+    }
+
+    if (title && date && contentful_id && summary && imgSrc) {
+      news.push(newNewsItem)
+    }
+
+    return news
+  }, [])
+
+  return (
+    <div className="news-archive-container">
+      {newsItems.map(item => {
+        return (
+          <NewsItem
```

Comment:
I want to create a slug to be used for individual news item links. I'm not sure if Contentful provides us anything like this out of the box. If they don't, you'll need to take the title and convert spaces to dashes, etc. to create one.

Reply:
I think we should be able to create slugs from the news item title from contentful. I'm going to save this for another PR though. Should be an easy one to add as long as there are no duplicate news item titles.